ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java

```diff
@@ -491,6 +497,33 @@ public void execute() {
     }
   }
 
+  private final class TakeSnapshot extends ViewOperation {
+    private Snapshot snapshot;
+    private Promise promise;
+    private File destFile;
+
+    public TakeSnapshot(int tag, Snapshot snapshot, File destFile, Promise promise) {
```

Comment:
Can you add a short Javadoc comment explaining this takes a screenshot? The name "snapshot" wasn't very obvious to me.